demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureToLegacyConverter.php

```diff
@@ -64,6 +64,8 @@ public function convertToLegacy($procedure): array
 
         if (null !== $procedureArray['settings']) {
             $procedureArray['settings'] = $this->entityHelper->toArray($procedureArray['settings']);
+            $procedureArray['phaseObject'] = $this->entityHelper->toArray($procedure->getPhaseObject());
```

Comment:
Yes, this is needed because it will be used to generate a diff to create a report entry on update a procedure.
Especially to generate the diff of the dates and keys of a procedure phase on update a procedure or on setup a automatic phase switch.